estate/__manifest__.py

```diff
@@ -0,0 +1,7 @@
+{
+    'name': 'Real Estate',
+    'application': True,
+    'author': 'Vauxoo',
+    'license': 'LGPL-3',
+    'depends': ['base']
```

Comment:
Are you running pre-commit-vauxoo? Keys and values should be in double quotes.

And add a trailing comma after a key, please. To improve readability.

```suggestion
    'depends': ['base'],
```